subprojects/docs/src/docs/release/notes.md

```diff
@@ -120,6 +120,14 @@ versionCatalogs.named("libs").findLibrary("assertj-core").ifPresent { assertjCor
 
 Check the [version catalog API](javadoc/org/gradle/api/artifacts/VersionCatalog.html) for all supported methods.
 
+<a name="configuration-cache"></a>
+### Configuration Cache improvements
+The configuration cache is a feature that significantly improves build performance by caching the result of the configuration phase and reusing this for subsequent builds.
+
+#### New API in `FileSystemOperations` to create `CopySpec` instances when no `Project` is available
+
+The `FileSystemOperations` service now has a [copySpec()](javadoc/org/gradle/api/file/FileSystemOperations.html#copySpec--) method for creating `CopySpec` instances in a [configuration cache](userguide/configuration_cache.html) friendly way.
+The new method allows you to create, configure and use `CopySpec` instances during the [execution phase](userguide/build_lifecycle.html#sec:build_phases).
```

Comment:
```suggestion
The new method allows you to create, configure, and use `CopySpec` instances during the [execution phase](userguide/build_lifecycle.html#sec:build_phases).
```

Reply:
It might be beneficial to show a small code snippet of `CopySpec` being used in the execution phase here.